src/test/java/org/fcrepo/camel/integration/FcrepoConstantsIT.java

```diff
@@ -47,5 +47,8 @@ public void testRdfLexicon() {
         assertEquals(RdfNamespaces.INDEXING, RdfLexicon.INDEXING_NAMESPACE);
         assertEquals(RdfNamespaces.RDF, RdfLexicon.RDF_NAMESPACE);
         assertEquals(RdfNamespaces.LDP, RdfLexicon.LDP_NAMESPACE);
+        assertEquals(RdfNamespaces.REPOSITORY + "ServerManaged", RdfLexicon.SERVER_MANAGED.toString());
```

Comment:
Likely this update will go away.
Also, this class is a unit test, not an integration test. It should probably be renamed and moved... in another PR.